app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
```

Comment:
Not really at the moment, but in CCCD we ended up with quite a few, mostly static pages linked in the footer and other places, and this way it is a bit more organised. I can change it back to HomeController.